sencha-workspace/SlateAdmin/app/model/Term.js

```diff
@@ -1,4 +1,5 @@
 /*jslint browser: true, undef: true *//*global Ext*/
+//TODO: perhaps this should be in model/course directory with Course, Department, and Section? (also corresponding Emergence classes)
```

Comment:
@BillClinton This model should probably find its way to the slate-core-data package soon. On the backend though Term is outside the courses namespace: https://github.com/SlateFoundation/slate/tree/master/php-classes/Slate

It should be a top-level model on the client side too as lots of features outside of courses are structured on terms.